app/views/patients/_menu_pledge_button.html.erb

```diff
@@ -1,13 +1,29 @@
-<% if not patient.pledge_sent? %>
-  <%= link_to submit_pledge_button,
-              submit_pledge_path(patient),
-              class: 'submit-pledge-button',
-              aria: { label: t('patient.menu.submit_pledge') },
-              remote: true %>
+<% if Config.practical_support_mode? %>
+  <% if not patient.marked_complete? %>
+    <%= link_to mark_complete_button,
+                mark_complete_path(patient),
+                class: 'submit-pledge-button', # reuse classes from submit pledge button
+                aria: { label: t('patient.menu.mark_complete') },
+                remote: true %>
+  <% else %>
+    <%= link_to mark_incomplete_button,
+                mark_complete_path(patient),
+                class: 'submit-pledge-button',
+                aria: { label: t('patient.menu.mark_incomplete') },
+                remote: true %>
+  <% end %>
 <% else %>
```

Comment:
same question here as with the button controller function - very similar code between in/complete and submit/cancel.

Reply:
I think reasonable to collapse this one if you like. Up to you!